app/lib/main.dart

```diff
@@ -1,9 +1,4 @@
 import 'package:flutter/material.dart';
+import 'humane_transport_app.dart';
 
 void main() => runApp(HumaneTransportApp());
-
-class HumaneTransportApp extends StatelessWidget {
```

Comment:
Why split this into another file?

Reply:
According to flutter best practice (separation of concern), main.dart should only contain void main

Reply:
I should freshen up on best practices then!